src/pages/_app.js

```diff
@@ -43,12 +50,23 @@ const MyApp = (props) => {
       <UserContextProvider>
         <MembersProvider>
           <SearchMemberProvider>
-            <div className={classNames.root}>
-              <div className={classNames.main}>
-                {loading && <Spinner />}
-                <Component {...pageProps} />
+            <KeyboardProvider value={initialvalue}>
+              <div
+                className={classNames.root}
+                role="button"
+                tabIndex={0}
+                onKeyDown={(e) => {
+                  if (e.altKey) {
+                    setIsOptionKey(true);
+                  }
+                }}
```

Comment:
when will this _key_ be set to false?

Reply:
that we will handle in each component separately

Reply:
why so?

Reply:
I will do as you suggested in our call